_authors/osci_2023/odina_s.markdown

```diff
@@ -0,0 +1,8 @@
+name: Odinahon Saydahmedova
+short_name: Odina
+twitter: <twitter handle without the @>
+github: Odinahon
+
```

Comment:
The formatting here needs to be exact, as it will be parsed as yaml.
```suggestion
---
name: Odinahon Saydahmedova
short_name: Odina
github: Odinahon
```